src/g3log/filesink.hpp

```diff
@@ -15,11 +15,11 @@ namespace g3 {
 
    class FileSink {
    public:
-      FileSink(const std::string &log_prefix, const std::string &log_directory);
+      FileSink(const std::string &log_prefix, const std::string &log_directory, const std::string &logger_id="g3log");
       virtual ~FileSink();
 
       void fileWrite(LogMessageMover message);
-      std::string changeLogFile(const std::string &directory);
+      std::string changeLogFile(const std::string &directory, const std::string &logger_id="g3log");
```

Comment:
You can remove the default argument since when calling this as a sink it has to be specified
i.e. change it to be:
```
std::string changeLogFile(const std::string &directory, const std::string &logger_id);
```